src/java/test/org/apache/zookeeper/server/NIOServerCnxnTest.java

```diff
@@ -68,5 +74,19 @@ public void testOperationsAfterCnxnClose() throws IOException,
         } finally {
             zk.close();
         }
+
+    }
+
+    @Test
+    public void testClientResponseStatsUpdate() throws IOException, InterruptedException, KeeperException {
+        try (ZooKeeper zk = createClient()) {
+            ProposalStats stats = serverFactory.getZooKeeperServer().serverStats().getClientResponseStats();
+            assertEquals("", -1, stats.getLast());
+
+            zk.create("/a", "test".getBytes(), Ids.OPEN_ACL_UNSAFE,
+                    CreateMode.PERSISTENT);
+
+            assertThat(stats.getLast(), greaterThan(0));
```

Comment:
0, looks like magic number. Any possibility to give a non-zero value, matching closer to the expected value ?

Also, please add error message - assertThat("error message", stats.getLast(), greaterThan(0));

Reply:
+1 for assert messages
Did you mean -1 as the magic number which means that stat has not been updated yet...?

Reply:
please add comments about the expectations like, 
assertEquals("stat should be initialized with -1", -1, stats.getLast());  or a better message:)

that will help easy test case maintenance.
b) greaterThan(0) , similarly good to add a message explaining why 0?